dj/api/nodes.py

```diff
@@ -722,6 +724,14 @@ def create_a_source(
 
     # Point the node to the new node revision.
     save_node(session, node_revision, node, data.mode)
+    make_history(
+        session,
+        History(
+            entity_type=EntityType.NODE,
+            entity_name=data.name,
+            activity_type=ActivityType.CREATE,
+        ),
+    )
```

Comment:
Here's a an example of adding a simple event to the history table that captures a node being created.